assets/Product.wxs

```diff
@@ -60,17 +60,27 @@
 
     <!--We need to show EULA, and provide option to customize download location-->        
     <Property Id="WIXUI_INSTALLDIR" Value="INSTALLFOLDER" />
-	
-    <!-- Prerequisites check for Windows Universal C time and Visual Studio 2015 C++ redistributables -->
+
+    <!-- Prerequisite check for Windows Universal C time -->
     <Property Id="UCRTINSTALLED" Secure="yes">
-      <DirectorySearch Id="Windows_System32" Path="[WindowsFolder]System32" Depth="0">
+      <DirectorySearch Id="Windows_System32_search1" Path="[WindowsFolder]System32" Depth="0">
         <FileSearch Name="ucrtbase.dll"/>
       </DirectorySearch>
     </Property>	
     <Condition Message="$(env.ProductName) requires the Universal C Runtime to be installed. You can download it here: https://www.microsoft.com/download/details.aspx?id=50410">
       <![CDATA[Installed OR UCRTINSTALLED]]>
     </Condition>
 
+     <!-- Prerequisite check for Visual Studio 2015 C++ redistributables -->
+     <Property Id="VCRUNTIMEINSTALLED" Secure="yes">
+      <DirectorySearch Id="Windows_System32_search2" Path="[WindowsFolder]System32" Depth="0">
+        <FileSearch Name="vcruntime140.dll"/>
+      </DirectorySearch>
+    </Property>	
+    <Condition Message="$(env.ProductName) requires the Visual Studio 2015 C++ redistributables to be installed. You can download it here: https://www.microsoft.com/download/details.aspx?id=48145">
```

Comment:
The link is not persistent and may be changed in any time. I think it is better remove it. Anybody can find it by means of Bing/Google.

Reply:
This is why there are tests to check that the links are still active. Also the original issue #4458 requested that it `'should give a good error message directing user to download'`

Reply:
I don't know how often those links get recycled, but it seems another solution would be to link back to our pre-requisites page: https://github.com/PowerShell/PowerShell/blob/master/docs/installation/windows.md#prerequisites

Reply:
Ok. Do you want me to link to the pre-requisites page then or do you want some time to think about it first? But if those links get recycled then the pre-requisites page will suffer from the same issue (except for that it can be updated even after release, which is not an insignificant advantage)

Reply:
@bergmeister the fact that we can update the pre-reqs page fairly easily and quickly is the advantage I was thinking

Reply:
@SteveL-MSFT Can PG request a permanent link for Universal C runtime?